zebra/zebra_rib.c

```diff
@@ -99,6 +99,7 @@ static const struct {
 	[ZEBRA_ROUTE_BABEL] = {ZEBRA_ROUTE_BABEL, 100, 2},
 	[ZEBRA_ROUTE_SHARP] = {ZEBRA_ROUTE_SHARP, 150, 4},
 	[ZEBRA_ROUTE_PBR] = {ZEBRA_ROUTE_PBR, 200, 4},
+	[ZEBRA_ROUTE_BFD] = {ZEBRA_ROUTE_BFD, 255, 4},
 
```

Comment:
not sure I understand relationship between route entry and BFD.
Is there a plan for BFD to insert routes?

Reply:
no there is no plan.  Hence the admin distance of 255.  The goal is to have a fully filled out route_info table so that there is no accidental usage.  I just wanted a uniformity of data and this table has all but BFD then we should add it with the understanding that it shouldn't happen.